### DISCUSSION TITLE
Access to request version 3 report for  Sponsored Brands

### DISCUSSION THREAD

**Discussion by Lin1031** [amzn/ads-advanced-tools-docs#180] [Q&A] [answered]
When the formal version or beta version launches，do we still need to submit a jira ticket if we want to access to request version 3 report for Sponsored Brands?

**Comment by elizafarley** [comment, marked as answer]
No, the allowlist is temporary. We are only gating the feature for now since it does not support legacy SB campaigns. Once that functionality is available, we will remove the allowlist.
